core/modules/qmeta/QStatusMysql.cc

```diff
@@ -0,0 +1,189 @@
+    //if (not _conn.runQuery(query, results, errObj)) { &&&
+    if (not _conn.runQuery(query, errObj)) {
+        LOGS(_log, LOG_LVL_ERROR, "SQL query failed: " << query);
+        // If this doesn't work, it is not vital to qserv functionality.
+        throw SqlError(ERR_LOC, errObj);
+    }
+
+    trans.commit();
+}
+
+
+void QStatusMysql::queryStatsTmpChunkUpdate(QueryId queryId, int completedChunks) {
+    std::lock_guard<std::mutex> sync(_dbMutex);
+    QMetaTransaction trans(_conn);
+    sql::SqlErrorObject errObj;
+    //sql::SqlResults results; &&&
+    std::string query = "UPDATE QStatsTmp SET completedChunks = ";
+    query += std::to_string(completedChunks);
+    query += ", lastUpdate = NOW() WHERE queryId = ";
+    query += std::to_string(queryId);
+
+    LOGS(_log, LOG_LVL_DEBUG, "Executing query: " << query);
+    // if (not _conn.runQuery(query, results, errObj)) {
+    if (not _conn.runQuery(query, errObj)) {
+        LOGS(_log, LOG_LVL_ERROR, "SQL query failed: " << query);
+        // If this doesn't work, it is not at all vital to qserv functionality.
+        throw SqlError(ERR_LOC, errObj);
+    }
+
+    trans.commit();
+}
+
+
+QStats QStatusMysql::queryStatsTmpGet(QueryId queryId) {
+    std::lock_guard<std::mutex> sync(_dbMutex);
+    QMetaTransaction trans(_conn);
+    sql::SqlErrorObject errObj;
+    sql::SqlResults results;
+    std::string query = "SELECT queryId, totalChunks, completedChunks, "
+                        "UNIX_TIMESTAMP(queryBegin), UNIX_TIMESTAMP(lastUpdate) "
+                        "FROM QStatsTmp WHERE queryId= ";
+    query += std::to_string(queryId);
+    LOGS(_log, LOG_LVL_DEBUG, "Executing query: " << query);
+    if (not _conn.runQuery(query, results, errObj)) {
+        LOGS(_log, LOG_LVL_ERROR, "SQL query failed: " << query);
+        throw SqlError(ERR_LOC, errObj);
+    }
+
+    sql::SqlResults::iterator rowIter = results.begin();
+    if (rowIter == results.end()) {
+        // no records found
+        throw QueryIdError(ERR_LOC, queryId);
+    }
+
+    // make sure that iterator does not move until we are done with row
+    sql::SqlResults::value_type const& row = *rowIter;
+
+    QueryId qId            = boost::lexical_cast<QueryId>(row[0].first);
+    int totalChunks        = boost::lexical_cast<int>(row[1].first);
+    int completedChunks    = boost::lexical_cast<int>(row[2].first);
+    std::time_t begin      = boost::lexical_cast<std::time_t>(row[3].first);
+    std::time_t lastUpdate = boost::lexical_cast<std::time_t>(row[4].first);
+
+    trans.commit();
+    return QStats(qId, totalChunks, completedChunks, begin, lastUpdate);
+}
+
+
+void QStatusMysql::queryStatsTmpRemove(QueryId queryId) {
+    std::lock_guard<std::mutex> sync(_dbMutex);
+    QMetaTransaction trans(_conn);
+    sql::SqlErrorObject errObj;
+    //sql::SqlResults results; &&&
+    std::string query = "DELETE FROM QStatsTmp WHERE queryId =";
+    query += std::to_string(queryId);
```

Comment:
These two lines could be a single line:
```
auto const query = "DELETE FROM QStatsTmp WHERE queryId =" + std::to_string(queryId);
```
The compiler would automatically make an object of type `std::string`